android/library/src/main/java/io/logto/android/Logto.kt

```diff
@@ -50,12 +51,36 @@ object Logto {
                     onComplete(error, null)
                 }
             }
-        ).startAuth(context)
+        ).login(context)
+    }
+
+    fun logoutWithBrowser(
+        context: Context,
+        onComplete: (error: Error?) -> Unit
+    ) {
+        checkInitState()
+        credential?.let {
+            BrowserLogoutFlow.init(
+                logtoConfig,
+                it.idToken,
+            ) {
+                error ->
+                if (error == null) {
```

Comment:
the indent looks weird. will the following work?
```suggestion
                error -> if (error == null) {
```